docs/modules/user-guide/partials/proc_adding-volume-components.adoc

```diff
@@ -0,0 +1,34 @@
+[id="proc_adding-volume_{context}"]
+= Adding `volume` components to a devfile
+
+[role="_abstract"]
+Use volume components in order to share files among container components, making it easier to collaborate with other teams during the development process.
```

Comment:
I wanted to give a `why` they'd care about sharing files across container components. What do y'all think of this `why I care` statement at the end of this abstract?

Reply:
This is very good information to provide. It speaks directly to the user story.